src/addremove.js

```diff
@@ -0,0 +1,73 @@
+// import Todo from './index.js';
```

Comment:
- [ ] Please remove commented pieces of code as they make the codebase messy and unprofessional. Use git history if you need to review a previous version of your codebase.